apps/costcontrol/test/unit/widget_startup_test.js

```diff
@@ -461,4 +453,94 @@ suite('Widget Startup Modes Test Suite >', function() {
     AirplaneModeHelper._status = 'disabled';
     Widget.init();
   });
+
+  test('DSDS start-up after SIM switching: from data usage only SIM to ' +
+         ' prepaid SIM',
```

Comment:
Align the start of the text with the text above and don't break after for the function callback, please.